persistence/sql/migrations/20241129111700000000_add_refresh_token_access_token_link.autocommit.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE hydra_oauth2_refresh ADD access_token_signature VARCHAR(255) DEFAULT NULL;

```

Comment:
Instead of storing the raw signature, wouldn't a foreign key make more sense?

Reply:
I did not see a use case for cascade here - we don't want to delete the refresh token when the access token is revoked during graceful refresh, otherwise it would no longer be active. So only "SET NULL" would be useful, but it's not really that useful and worth the overhead.